src/app/components/Nav/index.css

```diff
@@ -69,3 +69,7 @@ li > span {
 p {
   padding: 1rem;
 }
+
+.listLink {
+  line-height: 1em;
```

Comment:
This change prevents monitor names from collapsing.

### Before
<img width="738" alt="screen shot 2018-01-11 at 9 13 08 am" src="https://user-images.githubusercontent.com/7367/34837436-aa536508-f6af-11e7-8166-af46c6a4ad71.png">

### After
<img width="738" alt="screen shot 2018-01-11 at 9 11 48 am" src="https://user-images.githubusercontent.com/7367/34837409-9427645a-f6af-11e7-82d5-69098e04a65f.png">